packages/create-plugin/templates/common/_package.json

```diff
@@ -23,7 +23,8 @@
     "@grafana/e2e-selectors": "{{ grafanaVersion }}",{{/unless}}
     "@grafana/eslint-config": "^7.0.0",{{#if usePlaywright}}
     "@grafana/plugin-e2e": "^1.0.1",{{/if}}
-    "@grafana/tsconfig": "^1.2.0-rc1",{{#if usePlaywright}}
+    "@grafana/tsconfig": "^1.2.0-rc1",
+    "@grafana/plugin-meta-extractor": "^1.0.0",{{#if usePlaywright}}
```

Comment:
@leventebalogh now you've got a canary version of the meta-extractor, if you set this to `canary` you should end up with a canary verison of create-plugin that will allow you to check the output of both this PR and that the plugin-meta-extractor is working.

Reply:
Good idea! 💡 

Trying in https://github.com/grafana/plugin-tools/pull/871/commits/fb72a9412af072a323dae982cdda2a59c9f7eef4.

Reply:
Looks like it's working!